service/resource/cloudformation/resource.go

```diff
@@ -0,0 +1,89 @@
+package cloudformation
+
+import (
+	"github.com/aws/aws-sdk-go/service/cloudformation/cloudformationiface"
+	"github.com/giantswarm/awstpr"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+)
+
+const (
+	// Name is the identifier of the resource.
+	Name = "cloudformation"
+)
+
+type AWSConfig struct {
+	AccessKeyID     string
+	AccessKeySecret string
+	SessionToken    string
+	Region          string
+	accountID       string
+}
+
+// Config represents the configuration used to create a new config map resource.
+type Config struct {
+	// Dependencies.
+	AwsClient cloudformationiface.CloudFormationAPI
+	Logger    micrologger.Logger
+}
+
+// DefaultConfig provides a default configuration to create a new config map
+// resource by best effort.
+func DefaultConfig() Config {
+	return Config{
+		// Dependencies.
+		AwsClient: nil,
+		Logger:    nil,
+	}
+}
+
+// Resource implements the config map resource.
+type Resource struct {
+	// Dependencies.
+	awsClient cloudformationiface.CloudFormationAPI
+	logger    micrologger.Logger
+}
+
+// New creates a new configured config map resource.
+func New(config Config) (*Resource, error) {
+	// Dependencies.
+	if config.AwsClient == nil {
+		return nil, microerror.Maskf(invalidConfigError, "config.AwsClient must not be empty")
+	}
+	if config.Logger == nil {
+		return nil, microerror.Maskf(invalidConfigError, "config.Logger must not be empty")
+	}
+
+	newService := &Resource{
+		// Dependencies.
+		awsClient: config.AwsClient,
+		logger: config.Logger.With(
+			"resource", Name,
+		),
+	}
+
+	return newService, nil
+}
+
+func toCustomObject(v interface{}) (awstpr.CustomObject, error) {
```

Comment:
toCustomObject could be added to the key service. We've done that in newer operators.

Reply:
Done